Dockerfile

```diff
@@ -1,23 +1,38 @@
-FROM golang:1.20.14-alpine3.19
+FROM golang:1.20.14-alpine3.19 as build
 
-ARG GIT_COMMIT=unspecified
+ENV GOEXPERIMENT=arenas
+
+WORKDIR /opt/ceremonyclient
+
+COPY . .
+
+WORKDIR /opt/ceremonyclient/node
+
+RUN go install ./...
+RUN go install github.com/fullstorydev/grpcurl/cmd/grpcurl@latest
+
+FROM alpine:3.19
```

Comment:
both the build and the runtime stages must be based on the same linux distro, the other easy option is to base them on Debain

@CassOnMars can you see any issues with Alpine in production?

Reply:
Not to my knowledge, every platform has surprising idiosyncracies though. Eventually it'll make sense to build out a rumpkernel for the image

Reply:
thinking of this, not familiar with rumpkernels, but docker uses the host kernel as far as I know so this might not work with docker?